tests/by-util/test_tr.rs

```diff
@@ -1166,6 +1166,16 @@ fn check_against_gnu_tr_tests_empty_eq() {
         .stderr_is("tr: missing equivalence class character '[==]'\n");
 }
 
+#[test]
+fn check_too_many_chars_in_eq() {
+    // # Ensure that these fail.
```

Comment:
The comment is unnecessary because what it describes is obvious from looking at the code.
```suggestion
```